newton/examples/example_ik_benchmark.py

```diff
@@ -0,0 +1,426 @@
+    "h1": RobotCfg(
+        asset="h1_description",
+        file=Path("mjcf/h1.xml"),
+        parser=newton.utils.parse_mjcf,
+        ee_names=("left_hand", "right_hand", "left_foot", "right_foot"),
+        ee_links=(15, 19, 5, 10),
+        up_axis="Y",
+        seeds=256,
+        sub_batch_size=25,
+        lambda_factor=2.0,
+    ),
+    "franka": RobotCfg(
+        asset="franka_description",
+        file=Path("urdfs/fr3.urdf"),
+        parser=partial(newton.utils.parse_urdf, scale=1.0),
+        ee_names=("ee",),
+        ee_links=(9,),
+        up_axis="Z",
+        seeds=64,
+        sub_batch_size=1000,
+        lambda_factor=4.0,
+    ),
+}
+
+
+@lru_cache(maxsize=64)
+def _roberts_root(dim: int) -> float:
+    x = 1.5
+    for _ in range(20):
+        f = x ** (dim + 1) - x - 1.0
+        df = (dim + 1) * x**dim - 1.0
+        x_next = x - f / df
+        if abs(x_next - x) < 1.0e-12:
+            break
+        x = x_next
+    return x
+
+
+@wp.kernel
+def _pick_best(costs: wp.array(dtype=float), n_seeds: int, best: wp.array(dtype=int)):
+    q = wp.tid()
+    base = q * n_seeds
+    best_cost = float(1.0e30)
+    best_seed = int(0)
+    for s in range(n_seeds):
+        c = costs[base + s]
+        if c < best_cost:
+            best_cost, best_seed = c, s
+    best[q] = best_seed
+
+
+@wp.kernel
+def _scatter_winner(
+    src: wp.array2d(dtype=float),
+    winners: wp.array2d(dtype=float),
+    best: wp.array(dtype=int),
+    offs: wp.array(dtype=int),
+    n_seeds: int,
+    n_coords: int,
+):
+    sb_start = offs[0]
+    q = wp.tid()
+    src_idx = q * n_seeds + best[q]
+    for d in range(n_coords):
+        winners[sb_start + q, d] = src[src_idx, d]
+
+
+class Example:
+    def __init__(self, robot="h1", batch_sizes=(1, 10, 100, 1_000, 2_000), repeats=3):
+        self.robot_name = robot
+        self.batch_sizes = batch_sizes
+        self.repeats = repeats
+
+        self.iterations = 16
+        self.step_size = 1.0
+        self.pos_thresh_m = 5e-3
+        self.ori_thresh_rad = 0.05
+
+        self.device = "cuda" if wp.get_preferred_device().is_cuda else "cpu"
+        wp.set_device(self.device)
+
+        self.cfg = ROBOTS[robot]
+        self.model = self._create_robot()
+        self.n_coords = self.model.joint_coord_count
+
+        self.results = []
+        self.rng = np.random.default_rng()
+
+    def _create_robot(self) -> newton.Model:
+        builder = newton.ModelBuilder()
+        builder.num_rigid_contacts_per_env = 0
+        builder.default_shape_cfg.density = 100.0
+        builder.up_axis = self.cfg.up_axis
+
+        asset_path = (
+            Path(self.cfg.asset) / self.cfg.file
+            if ("/" in self.cfg.asset or "\\" in self.cfg.asset)
+            else newton.utils.download_asset(self.cfg.asset) / self.cfg.file
+        )
+        self.cfg.parser(asset_path, builder, floating=False)
+
+        model = builder.finalize(requires_grad=False)
+        model.ground = True
```

Comment:
You need to call `builder.add_ground_plane()` now, `model.ground` has been removed.